api.planx.uk/modules/send/utils/exportZip.ts

```diff
@@ -199,11 +200,11 @@ export class ExportZip {
   filename: string;
   private tmpDir: string;
 
-  constructor(sessionId: string) {
+  constructor(sessionId: string, flowSlug: string) {
     this.zip = new AdmZip();
     // make a tmp directory to avoid file name collisions if simultaneous applications
     this.tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), sessionId));
-    this.filename = path.join(__dirname, `ripa-test-${sessionId}.zip`);
+    this.filename = path.join(__dirname, `${flowSlug}-${sessionId}.zip`);
```

Comment:
This gives me the following on Pizza - 
<img width="748" alt="image" src="https://github.com/theopensystemslab/planx-new/assets/20502206/bba36fa0-9e63-4c8e-888c-88a30f219ab4">

Are you getting this also? Could we drop `dirname` here?

Reply:
`dirname` only occurs on docker envs as I understand it (so local & pizzas), should be automatically omitted on staging / prod !

Reply:
eg http://api.editor.planx.dev/admin/session/d4d9249a-dc1e-4996-9bfc-15fd3adab242/zip returns plain `ripa-test-{sessionId}.zip`

Reply:
Perfect, thanks for clarifying! Makes sense considering we don't get this on staging / prod.